src/super_gradients/training/transforms/keypoint_transforms.py

```diff
@@ -353,6 +386,14 @@ def __init__(
         self.mask_pad_value = mask_pad_value
         self.prob = prob
 
+    def __repr__(self):
+        return (
+            self.__class__.__name__
+            + "(max_rotation={0}, min_scale={1}, max_scale={2}, max_translate={3}, image_pad_value={4}, mask_pad_value={5}, prob={6})".format(
+                self.max_rotation, self.min_scale, self.max_scale, self.max_translate, self.image_pad_value, self.mask_pad_value, self.prob
+            )
+        )
```

Comment:
Similar question as above, I tend to think that `. format` is quite hard to read actually, especially with so many params, and only use it for "templates". Is there any reason for not using f-string ?
To make it clear I'm not asking to change, I think it does not matter here ✌️